src/components/TablatureBlock/TablatureBlock.js

```diff
@@ -0,0 +1,63 @@
+// Components / hooks
+import SimpleBar from "simplebar-react";
```

Comment:
why the change to SimpleBar and "blocks" instead of "bars"?

Reply:
SimpleBar is actually an NPM package that allows us to style the scrollbars. I have it implemented on cards where the tablatureBlock is longer than the card. The conflicting name is one reason I migrated from Bar to Block.

I also added functionality where users can add a blank text area to write notes in. Bar felt like 1 thing - a bar of tablature. It was easier to conceptualize these as blocks stacking on top of each other so I just went with block.  Blocks can have types, tablature or note, and blocks put together make up the whole document that the user creates.

Reply:
The note and tablature blocks are pushed into the same array on the backend - "blocks" which used to be bars -- and thats how the documents maintain their original order of things. But again I felt like a "bar" didn't accurately represent what was happening so I switched it up. Maybe just me being picky -- it all worked regardless of the name.